pkg/operation/seed/seed.go

```diff
@@ -803,31 +804,28 @@ func bootstrapComponents(c kubernetes.Interface, namespace string, imageVector i
 		return nil, err
 	}
 
-	// cluster-autoscaler
-	components = append(components, clusterautoscaler.NewBootstrapper(c.Client(), namespace))
-
 	// etcd
 	var (
 		etcdImage                string
+		schedulerImage           string
```

Comment:
This "block" was used for etcd only. Now we have scheduler and gsac artefacts here. Can you please separate this again for a better code overview?